exercises/practice/pangram/.docs/instructions.md

```diff
@@ -1,9 +1,8 @@
 # Instructions
 
-Determine if a sentence is a pangram.
-A pangram (Greek: παν γράμμα, pan gramma, "every letter") is a sentence using every letter of the alphabet at least once.
-The best known English pangram is:
+Your task is to figure out if a sentence is a pangram.
 
-> The quick brown fox jumps over the lazy dog.
+A pangram is a sentence using every letter of the alphabet at least once.
```

Comment:
Since we state "the alphabet" should we explicitly state what that alphabet is?  We might enumerate the letters below, rather than an implied range.